src/domain/matomo/matomo.tsx

```diff
@@ -0,0 +1,43 @@
+import {
```

Comment:
Looks like version control may have this file name in lowercase.

Reply:
Fixed casing with `git mv`. Seems like it is not changed just by changing it from vscode, exprienced this problem in the past too.